patches/parse+3.2.0.patch

```diff
@@ -0,0 +1,11 @@
+diff --git a/node_modules/parse/lib/react-native/EventEmitter.js b/node_modules/parse/lib/react-native/EventEmitter.js
```

Comment:
I don't think this file should be included

Reply:
It should, ignore this comment!

Reply:
Its a known issue, it is this patch:
https://github.com/microsoft/react-native-windows/issues/7451#issuecomment-856392511